src/accelerate/utils/imports.py

```diff
@@ -90,20 +97,19 @@ def is_cuda_available():
 
 
 @lru_cache
-def is_tpu_available(check_device=True):
```

Comment:
We also should have a deprecation warning for `is_tpu_available` here as well that points to `is_torch_xla_available`. Deprecations should be two `0.x.0` versions

Reply:
Done. I have added deprecation warnings for both `is_tpu_available` and `DistributedType.TPU`.